Documentation/Books/Manual/Downgrading/README.md

```diff
@@ -0,0 +1,64 @@
+Downgrading
+===========
+
+A direct, binary, downgrade of ArangoDB is **not** supported. If you have upgraded
+your ArangoDB package, and then also upgraded your current data directory, it is
+not supported to downgrade the package and start an older ArangoDB version on a
+data directory that was upgrade already.
+
+If you are using a standalone ArangoDB server, data directory could have been upgraded
+automatically during package upgrade. If you are using the _Starter_ to start your
+ArangoDB, and you have not triggered yet the rolling upgrade, or upgraded it
+manually, your data directory is (probably) still on the old version, so you should
+be able to binary downgrade in this case.
+
+Supported Downgrade Procedures
+------------------------------
+
+In order to downgrade, the following options are available:
+
+- Restore a backup you took using the tool [Arangodump](../Programs/Arangodump/README.md)
+  before the upgrade.
+- Start the old package on the data directory backup you took before the upgrade.
+
+### Restore an _arangodump_ Backup
+
+This procedure assumes that you have taken an _arangodump_ backup using the old
+ArangoDB version, before you upgraded it. 
+
+1. Stop ArangoDB (if you are using an Active Failover, or a Cluster, stop all the needed
+   processes on all the machines).
+1. As extra precaution, take a backup of your current data directory (at filesystem level).
+   If you are using an Active Failover or a Cluster, you will need to backup all the data
+   directories of all the processes involved, from all machines. Make sure you move your
+   data directory backup to a safe place.
+1. Uninstall the ArangoDB package (use appropriate _purge_ option so your current data
+   directory is deleted, if you are using a stand alone instance).
+1. Install the old version of ArangoDB.
+1. Start ArangoDB. If you are using the _Starter_, please make sure you use a new data
+   directory for the _Starter_.
+1. Restore the _arangodump_ backup taken before upgrading.
```

Comment:
Markdown is supposed to be readable in plaintext as well, but allows this pattern for ordered lists. I would prefer to use sequential numbers in the source however, like

```
1. foo
2. bar
3. baz
```

Reply:
done.